erpnext/stock/doctype/item_barcode/item_barcode.json

```diff
@@ -25,7 +25,7 @@
    "fieldtype": "Select",
    "in_list_view": 1,
    "label": "Barcode Type",
-   "options": "\nEAN\nUPC-A"
```

Comment:
This will invalidate existing UPC-A barcodes. Write a thin compatibility layer to translate ERPNext types and barcode library types. Can be a simple constant dictionary in file.